src/style/app.module.css

```diff
@@ -603,16 +615,48 @@
 
 .addButton {
   margin-bottom: 10px;
-  color: var(--brown-color);
+  color: var(--dark-grey-font);
   background-color: var(--light-blue);
   border-color: var(--grey-bg-color);
+  --bs-btn-focus-box-shadow: none;
 }
 
-.addButton:is(:hover, :active) {
+.addButton:is(:hover, :active, :focus-visible) {
   background-color: var(--dark-blue) !important;
   border-color: var(--search-button-border);
 }
 
+.addButton:disabled {
+  margin-bottom: 10px;
+  background-color: var(--disabled-btn);
+  border-color: var(--grey-bg-color);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`